aws-yaml-ci.yml

```diff
@@ -0,0 +1,177 @@
+# Variable 'AWS_ACCESS_KEY_ID' defined in the Variable Group tab
```

Comment:
What purpose do these comments serve?

Reply:
I know that. The question is why have you left them in! What is your purpose in leaving them in?